lucene/queries/src/java/org/apache/lucene/queries/intervals/MultiTermIntervalsSource.java

```diff
@@ -96,7 +96,7 @@ public MatchesIterator matches(String field, LeafReaderContext ctx, int doc) thr
 
   @Override
   public void visit(String field, QueryVisitor visitor) {
-    visitor.visitLeaf(new IntervalQuery(field, this));
+    automaton.visit(visitor, new IntervalQuery(field, this), field);
```

Comment:
eh; what's this about?

Reply:
It's the intervals equivalent of AutomatonQuery - it means that Intervals.multiterm() should start working with the highlighter